lms/djangoapps/instructor/lti_grader.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+"""
+Send grades to an OpenEdX LTI component
+"""
+import csv
+
+from instructor import lti_connection
+from instructor.lti_connection import LTIConnectionError
+
+
+class LTIGrader(object):
+    """
+    Updates grades for a given LTI component
+    """
+
+    def __init__(self, course_id, url_base, lti_key, lti_secret):
+        self.url_base = url_base
+        self.course_id = course_id
+        self.key = lti_key
+        self.secret = lti_secret
+        self.output = {'success': '', 'error': ''}
+
+    def _get_first_anon_id(self, grade_data):
+        """
+        Returns the first anonymized student id for in the given uploaded csv data file
+        :param grade_data: an InMemoryUploadedFile with lti grade data
+        :return: anon_id
+        """
+        reader = self._unicode_csv_reader(grade_data)
+        try:
+            reader.next()
```

Comment:
Why are we discarding the first row?

I'd guess that's the header we're throwing away? If so, let's comment as much here. As is, it's not immediately apparent what the underlying data looks like.